test/cpp/jit/test_gpu_view.cpp

```diff
@@ -302,7 +301,6 @@ TEST_F(NVFuserTest, FusionViewReductionShmoo_CUDA) {
       {{1, 333}, {1, 1, 1, 111, 1, 3}},
       {{22, 1, 22, 1}, {484}},
       {{1, 333, 1}, {333}},
-      // Incorrect Result - Broadcast Issue - Reduction
```

Comment:
This test case was an issue when view is handled by the reduction scheduler. It passes right now because we only handle view through the pointwise scheduler.